test/constructorio-synonyms-one-way.js

```diff
@@ -0,0 +1,587 @@
+      const constructorio = new Constructorio({
+        apiToken: 'bad-token',
+        apiKey: 'bad-key',
+      });
+
+      constructorio.modifyOneWaySynonym({
+        phrase: oneWaySynonymPhrase,
+        child_phrases: [
+          { phrase: 'loaf' },
+          { phrase: 'buns' },
+          { phrase: 'toast' },
+        ],
+      }, (err, response) => {
+        expect(err).to.be.an('object');
+        expect(err).to.have.property('message').to.match(/You have supplied an invalid/);
+        expect(response).to.be.undefined;
+        done();
+      });
+    });
+  });
+
+  describe('getOneWaySynonyms', () => {
+    const oneWaySynonymPhrases = ['spices', 'utensils', 'snacks'];
+
+    before((done) => {
+      const addPromiseList = oneWaySynonymPhrases.map(addTestOneWaySynonym);
+
+      // Create test one way synonyms for use in tests
+      Promise.all(addPromiseList).then(() => done()).catch(() => done());
+    });
+
+    after((done) => {
+      const removePromiseList = oneWaySynonymPhrases.map(removeTestOneWaySynonym);
+
+      // Clean up test one way synonyms
+      Promise.all(removePromiseList).then(() => done()).catch(() => done());
+    });
+
+    it('should retrieve a listing of all one way synonyms', (done) => {
+      const constructorio = new Constructorio(testConfig);
+
+      constructorio.getOneWaySynonyms({}, (err, response) => {
+        expect(err).to.be.undefined;
+        expect(response).to.be.an('object');
+        expect(response).to.have.property('one_way_synonym_relations').to.be.an('array').length(3);
+        done();
+      });
+    });
+
+    it('should retrieve a listing of one way synonyms when supplying a phrase property', (done) => {
+      const constructorio = new Constructorio(testConfig);
+
+      // Note: Result set is not being checked for match as groups can take many seconds to be indexed / returned
+      constructorio.getOneWaySynonyms({
+        phrase: 'spices',
+      }, (err, response) => {
+        expect(err).to.be.undefined;
+        expect(response).to.be.an('object');
+        expect(response).to.have.property('one_way_synonym_relations').to.be.an('array');
+        done();
+      });
+    });
+
+    it('should return no results when supplying a non-existent phrase property', (done) => {
+      const constructorio = new Constructorio(testConfig);
+
+      constructorio.getOneWaySynonyms({
+        phrase: 'mallorca',
+      }, (err, response) => {
+        expect(err).to.be.undefined;
+        expect(response).to.be.an('object');
+        expect(response).to.have.property('one_way_synonym_relations').an('array').length(0);
+        done();
+      });
+    });
+
+    it('should retrieve a listing of one way synonyms when supplying num results per page property', (done) => {
+      const constructorio = new Constructorio(testConfig);
+
+      // Note: Result set is not being checked for match as groups can take many seconds to be indexed / returned
```

Comment:
Is this comment still true? If so, we should change "groups" to "synonyms".

Reply:
Yup, updated!